crates/yas_core/src/contracts/yas_pool.cairo

```diff
@@ -713,6 +737,10 @@ mod YASPool {
             Position::unsafe_new_contract_state()
         }
 
+        fn get_slot_0(self: @ContractState) -> Slot0 {
+            self.slot_0.read()
+        }
+
```

Comment:
Remove this function since it's the same as `slot_0`. In the tests that use this function, replace it with `slot_0`